kubernetes/deployment.yml

```diff
@@ -17,8 +16,8 @@ spec:
         app: prometheus-data-generator
     spec:
       containers:
-      - image: littleangryclouds/prometheus-data-generator:0.1
-        imagePullPolicy: Always
+      - image: quay.io/kaszpir/prometheus-data-generator:fixes
```

Comment:
Would you change it to `littleangryclouds/prometheus-data-generator:0.2`? At the moment of merging I'll generate the image and push it.

Reply:
Sure thing, will do.